contracts/validatorset/ValidatorSetRelayed.sol

```diff
@@ -164,18 +164,18 @@ contract ValidatorSetRelayed is IValidatorSetRelayed, Ownable {
         onlyRelay
     {
         finalized = true;
-        
-        for (uint256 i = 0; i < migrationValidators.length; i++) {
-            AddressStatus storage vstatus = addressStatus[migrationValidators[i]];
-            vstatus.state = ValidatorState.Finalized;
-            vstatus.index = i;
-        }
 
-        if (toBeRemoved != address(0)) {
+        if (toBeRemoved == address(0)) {
+            // The new added validator is always the last of `migrationValidators` array
```

Comment:
Comment could be improved. Can we explicitly write in the comment that this if-else branch is respectively adding / removing a validator?